src/org/zaproxy/zap/extension/quickstart/resources/help/contents/quickstart.html

```diff
@@ -21,11 +21,30 @@
 	For a more in depth test you should explore your application using your browser or 
 	automated regression tests while proxying through ZAP.
 </p>
+<H2>Launch Browser</H2>
+As long as you have the Selenium add-on installed then you will see an option to launch browsers.<br>
+Select the browser you want from the pull-down and press the 'Launch Browser' button.<br>
+Browsers launched this way will be configured to proxy via ZAP and will ignore certificate errors,
```

Comment:
Worth mentioning that the browser will stop "working" if the address/port of the Local Proxy is changed?